### DISCUSSION TITLE
Installation Second Kibana

### DISCUSSION THREAD

**Discussion by cherryCokeBack** [Security-Onion-Solutions/securityonion#12919] [2.4]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

12

### RAM

64

### Storage for /

2to

### Storage for /nsm

20to

### Network Traffic Collection

span port

### Network Traffic Speeds

more than 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

Hello, 
I'm looking to install a new kibana server in addition to the one already in place on the master server.

I've installed a new server with only Kibana version 8.10.4 to maintain compatibility with the elastic cluster

I've copied the kibana.yml configuration from the master server but I'm left with a blank page for kibana

if you have a solution ? 

Many thanks ! 

[KIBANALOG.txt](https://github.com/Security-Onion-Solutions/securityonion/files/15198671/KIBANALOG.txt)

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by InfosecGoon** [comment]
If you want to access Elasticsearch from a machine outside of the Security Onion grid, you'll need to add its IP to the elasticsearch-rest firewall group in Configuration.
